descriptors/ModuleDescriptor-template.json

```diff
@@ -95,6 +95,37 @@
         }
       ]
     },
+    {
+      "id": "organizations-storage.privileged-contacts",
+      "version": "4.7",
```

Comment:
version should be 1.0
It is version of API, not module

Reply:
and why do we have 2.0 for contacts, what's the difference?